pkg/fleet/installer/exec/dotnet_library_exec.go

```diff
@@ -0,0 +1,89 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build windows
+
+package exec
```

Comment:
nit: put this in a pacakge under `installer/packages`, this `exec` package is for the CLI client of the installer